### ISSUE TITLE
Gcop 538

### ISSUE THREAD

**Issue by Karvan** [Geeksltd/GCop#8]
Gcop 538 is showing false alarm.

code: `if(Account.URLExtension.IsValidURL() == false) `
warning:  GCop538	File extensions can have different casing. Use case insensitive string comparison.

**Comment by Foroughi** [comment]
@reporter, The same idea that we used for GCope537 (which caused issue #10) is being used for GCope538. So I'm gonna do the same scenario I used to fix #10, for this issue as well. 

1. Specifying the exact behavior we expect from GCope539
2. Start from scratch and recode the GCope539

What we expect from GCope538 is to kick in under the following circumstances : 

A statement with the following condition :
1 - Equals Expression (== operator)
2 - one side is a Literal Expression Syntax (a hard-coded string)
3 - the other side is System.IO.FileInfo.Extension

If we have all these conditions in one statement, we need to inform the user to use another syntax like : 

`new System.IO.File("File address").Extension.ToLower() == ".ext".ToLower()`
